source/draw/RasterMonochrome.ooc

```diff
@@ -224,4 +224,107 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		result
 	}
+	// Serialize to a lossy ascii image using an alphabet
+	// Precondition: alphabet may not have extended ascii, non printable, '\', '"', '>' or linebreak
+	// Example alphabet: " .,-_':;!+~=^?*abcdefghijklmnopqrstuvwxyz()[]{}|&%@#0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	toAscii: func (alphabet: String) -> String {
```

Comment:
Ah we can look at it later if someone cares, let's go with `String` right now as you've done.